python/pylibraft/pylibraft/cluster/kmeans.pyx

```diff
@@ -84,37 +84,40 @@ def compute_new_centroids(X,
     Examples
     --------
 
-    >>> import cupy as cp
+    .. code-block:: python
 
-    >>> from pylibraft.common import Handle
-    >>> from pylibraft.cluster.kmeans import compute_new_centroids
+        >>> import cupy as cp
 
```

Comment:
```suggestion
```
@lowener To slove the issue you are seeing in the docs, you just have to delete these empty lines instead of creating a python code-block.. Good catch!

Reply:
One last argument that I have in favor of the code-block is that it improves readability. In the previous example there is small breaks between the includes, the variable declaration, and the RAFT code. I think it can be appreciated by our Python users. What do you think?

Reply:
IIUC, the .. code block is a sphinx-specific documentation style while the other format is more universal across python. Is that true?

Reply:
From what I understand as well yes. When running `?cagra.build` in IPython for details about this function users would see written literally `.. code-block::python`.

Reply:
Yeah, that's right. Lets drop `code-block` usages in that case.